src/modules/UI/components/PasswordReminderModal/components/PasswordInput.ui.js

```diff
@@ -0,0 +1,135 @@
+  labelHeight?: number,
+  labelPadding?: number,
+  inputContainerStyle?: Object,
+  inputContainerPadding?: number,
+  lineWidth?: number,
+  activeLineWidth?: number,
+  disabledLineWidth?: number,
+  title?: string,
+  prefix?: string,
+  suffix?: string,
+  error?: string,
+  errorColor?: string,
+  disabledLineType?: string,
+  animationDuration?: number,
+  characterRestriction?: Array<string>,
+  disabled?: boolean,
+  editable?: boolean,
+  multiline?: boolean,
+  containerStyle?: Object,
+  labelTextStyle?: Object,
+  titleTextStyle?: Object,
+  affixTextStyle?: Object,
+  renderAccessory?: boolean,
+  secureTextEntry?: boolean,
+  onChangeText?: Function,
+  onFocus?: Function,
+  onBlur?: Function,
+  value?: Function
+}
+export type State = {}
+
+export class PasswordInput extends Component<Props, State> {
+  render () {
+    const {
+      secureTextEntry,
+      tintColor,
+      baseColor,
+      label,
+      inputContainerStyle,
+      textColor,
+      fontSize,
+      titleFontSize,
+      labelFontSize,
+      labelHeight,
+      labelPadding,
+      inputContainerPadding,
+      lineWidth,
+      activeLineWidth,
+      disabledLineWidth,
+      title,
+      prefix,
+      suffix,
+      error,
+      errorColor,
+      disabledLineType,
+      animationDuration,
+      characterRestriction,
+      disabled,
+      editable,
+      multiline,
+      containerStyle,
+      labelTextStyle,
+      titleTextStyle,
+      affixTextStyle,
+      renderAccessory,
+      onChangeText,
+      onFocus,
+      onBlur,
+      value
+    } = this.props
+    return (
+      <TextField
+        secureTextEntry={secureTextEntry || DEFAULTS.secureTextEntry}
+        tintColor={tintColor || DEFAULTS.tintColor}
+        baseColor={baseColor || DEFAULTS.baseColor}
+        label={label || PASSWORD_TEXT}
+        inputContainerStyle={inputContainerStyle}
+        textColor={textColor}
+        fontSize={fontSize}
+        titleFontSize={titleFontSize}
+        labelFontSize={labelFontSize}
+        labelHeight={labelHeight}
+        labelPadding={labelPadding}
+        inputContainerPadding={inputContainerPadding}
+        lineWidth={lineWidth}
+        activeLineWidth={activeLineWidth}
+        disabledLineWidth={disabledLineWidth}
+        title={title}
+        prefix={prefix}
+        suffix={suffix}
+        error={error}
+        errorColor={errorColor}
+        disabledLineType={disabledLineType}
+        animationDuration={animationDuration}
+        characterRestriction={characterRestriction}
+        disabled={disabled}
+        editable={editable}
+        multiline={multiline}
+        containerStyle={containerStyle}
+        labelTextStyle={labelTextStyle}
+        titleTextStyle={titleTextStyle}
+        affixTextStyle={affixTextStyle}
+        renderAccessory={renderAccessory}
+        onChangeText={onChangeText}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        value={value}
+      />
+    )
+  }
+}
```

Comment:
Again: not broken. but why not use existing FormFieldinput component. If extra parameters were needed for the api, modifying and expanding on an existing single component.